include/boost/math/tools/color_maps.hpp

```diff
@@ -0,0 +1,156 @@
+//  (C) Copyright Nick Thompson 2021.
+//  (C) Copyright Matt Borland 2022.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_MATH_COLOR_MAPS_HPP
+#define BOOST_MATH_COLOR_MAPS_HPP
+
+#include <cmath>
+#include <array>
+#include <algorithm>
+#include <utility>
+#include <boost/math/tools/concepts.hpp>
+#include <boost/math/ccmath/floor.hpp>
+
+namespace boost::math::tools {
+
+namespace detail {
+
+template <BOOST_MATH_FLOATING_POINT_TYPE Real>
+class color_map
+{
+private:
+    std::array<Real, 256> r;
```

Comment:
generally the convention is the member variables have the underscore, e.g. `std::array<Real, 256> r_;`